### DISCUSSION TITLE
Please add install function

### DISCUSSION THREAD

**Discussion by Rohit24923** [tyron12233/CodeAssist#334] [Ideas]
**Feature** request to you add install function in CodeAssist. For example: When /storage/emulated/0/Details/Code Assist/Cool Hint/app/build/bin/signed.apk clicked on CodeAssist app not shows install option. Please add  install function in CodeAssist.
![Screenshot_20220316-055626](https://user-images.githubusercontent.com/100986884/158493532-8e67abbf-945f-4d5c-b7e1-2af596d782f6.png)

_Originally posted by @Rohit24923 in https://github.com/tyron12233/CodeAssist/issues/325#issuecomment-1068592623_

**Comment by susandahal** [comment]
> 

There already is an option to install apk in logs after you build. You can also enable option to get prompt to install app after build completes
